docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -358,6 +359,58 @@ export default function App() {
 }
 ```
 
+### Sharing a database between apps/extensions (iOS)
+
+To share a database with other apps/extensions in the same App Group you can use shared containers. Follow these steps:
+
+<Step label="1">
+Configure the App Group in `app.json`:
+```json app.json
```

Comment:
````suggestion
<Step label="1">

Configure the App Group in app config:

```json app.json
````

Nit: Using line breaks like the above here helps us to avoid any unknown markup rendering issues.
Let's change `app.json` to app config since this file can have different file formats in a project (example: app.config.js).